pisa/__init__.py

```diff
@@ -11,7 +11,14 @@
 import warnings
 
 from numba import jit as numba_jit
-from numba import NumbaDeprecationWarning
+
```

Comment:
All of these changes should be unnecessary because we require numba 0.45.1. That said, you can do the following (and get rid of all the "FOUND..." stuff):
```python
try:
    # numba 0.45.1
    from numba.errors import NumbaDeprecationWarning
except ImportError:
    # latest numba versions
    from numba.core.errors import NumbaDeprecationWarning
```
and that should be compatible with Numba versions newer than 0.45.1 (of course `SmartArray`s go away)

Reply:
I am using numba 0.43.1 which is where I have the error, and it looks like `NumbaDeprecationWarning` doens't exist there. I remember there was some issue with the default numba which is why I went to an older one but can't remember now what tat was (might have been a llvmlite version issue). But sounds like this is specific to non-standard stuff in my install then, so probably best just to drop this change.